src/frameworks/svelte-kit.json

```diff
@@ -9,7 +9,7 @@
   },
   "dev": {
     "command": "svelte-kit dev",
-    "port": 3000,
+    "port": 5173,
```

Comment:
@ascorbic, for svelte-kit, I know @brittneypostma ran into this the other day, so I wonder if it's OK to do the port upgrade to the new vite default port and for folks who may be using the old port, just update it? The error should be fairly clear when the CLI times out waiting for a server that is not running on port 3000.

Reply:
Seeing this comment after my other one, so there is a timeout I'm just too impatient to wait for it 😅